drivers/include/saul.h

```diff
@@ -279,6 +279,16 @@ typedef struct {
     uint8_t type;           /**< device class the device belongs to */
 } saul_driver_t;
 
+/**
+ * @brief   Initialize all available SAUL drivers.
+ *          This function is called automatically if the auto_init_saul
+ *          module is used.
+ *          If only the saul_init module is used instead, you can call
+ *          this function to manually set up all SAUL sensors at a later
+ *          time.
+ */
+void gnrc_netif_auto_init(void);
```

Comment:
```suggestion
void saul_auto_init(void);
```

Reply:
Why not `saul_init()` btw? Or `saul_init_devs()`? `saul_auto_init()` is misleading, now that the function can also be called manually. Moreover, the name is not really in line with what the brief description says (`Initialize all available SAUL drivers.` <=> `saul_auto_init()`).